src/pages/tabbed-dashboard/App.jsx

```diff
@@ -49,26 +49,51 @@ const App = () => {
   const chatWidgetRef = useRef();
   /** @type {React.MutableRefObject<Callbridge.Dashboard>} */
   const widgetRef = useRef();
+  /** @type {React.MutableRefObject<HTMLSelectElement>} */
+  const selectRef = useRef();
 
   const credentials = useSelector(state => state.credentials);
 
   const renderHideDashboardElementsButton = () => {
     const onHideDashboardElementsClick = () => {
       // Contact our support team on how to see the list of hidable elements
-      setHideDashboardElements([50,51,52,53])
+      setHideDashboardElements(hideDashboardElements ? undefined : [50, 51, 52, 53]);
     }
 
     return (
-      <button
-        type="button"
-        className={`${MenuButtonStyles.menuButton} ${MenuButtonStyles.right}`}
-        style={{ top: '130px' }} // why is this hardcoded?
-        onClick={onHideDashboardElementsClick}
-      >
-        Hide Dashboard Elements
-      </button>
+      <div>
+        <button
+          type="button"
+          className={`${MenuButtonStyles.menuButton} ${MenuButtonStyles.hideDashboardElementsButton} ${MenuButtonStyles.right}`}
+          onClick={onHideDashboardElementsClick}
+        >
+          {`${hideDashboardElements ? 'Show' : 'Hide'} Dashboard Elements`}
+        </button>
+      </div>
     )
   }
+  
+  const renderMultiSelect = () => {
+    const onMultiSelectChange = (event) => {
+      const options = Array.from(event.target?.options);
+      const selectedValues = options
+      .filter(option => option.selected)
+      .map(option => Number(option.value));
+  
+      setHideDashboardElements(selectedValues.length > 0 ? selectedValues : undefined);
+    }
+
+    return (
+      <div>
+        <select multiple onChange={onMultiSelectChange} ref={selectRef} className={`${MenuButtonStyles.menuButton} ${MenuButtonStyles.multiSelectHideDashboardElements} ${MenuButtonStyles.right}`}>
+          <option value={50}>50</option>
+          <option value={51}>51</option>
+          <option value={52}>52</option>
+          <option value={53}>53</option>
```

Comment:
i would think `hideDashboardElements.map((value) => (<option key value selected>))` is easier.

Reply:
That might have issues where if nothing is hidden then `hideDashboardElements` will be undefined and the multi-select will not render anything.

However I've refactored it to the below, since it makes more sense to have the select be changed/rendered based on state and not an `onChange`
```
{selectableHiddenElements.map(value => (
  <option key={value} value={value} selected={hideDashboardElements ? hideDashboardElements.includes(value) : false}>
    {value}
  </option>
))}
```

Reply:
react's select supports using an array for `value` https://react.dev/reference/react-dom/components/select